docs/configuration/actions.md

```diff
@@ -123,6 +124,40 @@ actions:
   - bash -c "sed -i -r \"s/Release:(\s*)\S+/Release:\1${PACKIT_RPMSPEC_RELEASE}%{?dist}/\" my_specfile_path"
 ```
 
+### `commit-message`
+
+For your own debugging purposes we allow arbitrary output before outputting the
+actual commit message. Your commit message **needs** to be separated by the
+following separator (on a separate line):
+
+    ---%<--- snip ---%<--- here ---%<---
```

Comment:
why not scissors? :(

`8<`

Reply:
it's almost the same :)

Reply:
btw I'm willing to change it, if you have a font that provides some fancy ligature for this :P